react/index.tsx

```diff
@@ -115,22 +71,10 @@ const OrderPlaced: FC = () => {
 
           <ExtensionPoint id="order-placed-top" orderGroup={orderGroup} />
 
-          <header>
-            <ExtensionPoint id="order-placed-confirmation" />
-            <ExtensionPoint id="order-placed-notices" />
-            <ExtensionPoint id="order-placed-summary" />
-            <ExtensionPoint id="order-placed-bank-invoice" />
-          </header>
+          <ExtensionPoint id="op-header" />
 
           <div className="mv6 w-80-ns w-90 center">
```

Comment:
Would add a CSS handle for this `<div>`

Reply:
Maybe even in the wrapping `<article>`

Reply:
Done!

Reply:
Changed the article to `<div>` and the `<div>` to `<main>`